core/src/common/lwm2m_plaintext.c

```diff
@@ -291,10 +291,10 @@ static int PTDeserialiseResource(SerdesContext * serdesContext, Lwm2mTreeNode **
 // Map PlainText serdes function delegates
 const SerialiserDeserialiser plainTextSerDes =
 {
-    .SerialiseObject   = NULL,
-    .SerialiseObjectInstance = NULL,
-    .SerialiseResource = PTSerialiseResource,
-    .DeserialiseObject = NULL,
-    .DeserialiseObjectInstance = NULL,
-    .DeserialiseResource = PTDeserialiseResource,
+    NULL,
```

Comment:
This is due to an incompatibility between C and C++. ISO C++ does not support this.